int.ts

```diff
@@ -1786,7 +1808,16 @@ module J2ME {
                       // handles.
                       address = ref[--sp];
                       if (typeof address === "number") {
-                        args.unshift(getHandle(address));
+                        if (address === 0) {
+                          args.unshift(null);
+                        } else {
+                          var tmp = getArrayFromAddr(address);
+                          if (tmp) {
+                            args.unshift(tmp);
+                          } else {
+                            args.unshift(getHandle(address));
+                          }
```

Comment:
Done in https://github.com/marco-c/pluotsorbet/commit/31af65155f995cc3b044e7caf0ea39991351bd2a.